TwitchChatVotingProxy/VotingReceiver/EVotingReceiver.cs

```diff
@@ -5,5 +5,6 @@ enum EVotingReceiver
     {
         Twitch,
         YouTube,
```

Comment:
If this proxy is added, we might want to rename the current (my) implementation to `YouTubeV3`, but that's not important right now.